src/App.jsx

```diff
@@ -1,51 +1,71 @@
+import { useState, useEffect } from "react";
```

Comment:
I prefer browserRouter implementations over hash routers, we discussed how you can fix this in the morning standup sessions